tests/integration/api_build_test.py

```diff
@@ -339,7 +339,6 @@ def test_build_with_extra_hosts(self):
 
         assert self.client.inspect_image(img_name)
         ctnr = self.run_container(img_name, 'cat /hosts-file')
-        self.tmp_containers.append(ctnr)
```

Comment:
I haven't dived into the code but why this change?

Reply:
The `self.run_container` already appends it to the list so there is no need to do it after.